holmes/config.py

```
@@ -649,20 +656,61 @@ def load_from_file(cls, config_file: Optional[str], **kwargs) -> "Config":
+                "provide a model_list.yaml file, or ensure ROBUSTA_AI is not disabled."
             )
-            api_key = model_params.pop("api_key", api_key)
-            model = model_params.pop("model", model)
 
-        return DefaultLLM(model, api_key, model_params)  # type: ignore
+        if not selected_model:
+            raise ValueError("Could not determine an LLM model to use.")
+
+        return DefaultLLM(selected_model, selected_api_key, selected_params)
+
+    def load_robusta_api_key(self, dal: SupabaseDal):
```

Ruff (0.8.2)

Missing return type annotation for public function `load_robusta_api_key`

Add return type annotation: `None`

(ANN201)